packages-tests/Caching/Detector/ChangedFilesDetectorTest.php

```diff
@@ -32,7 +32,7 @@ public function testHasFileChanged(): void
         $filePath = __DIR__ . '/Source/file.php';
```

Comment:
why are these test-changes required?

Reply:
because previous logic is splitted in 2 methods now, I renamed one so maybe it's clearer